File: CS/WindowsApplication1/Custom menu/DXMenuItemWrapper.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraEditors;
using DevExpress.Utils.Menu;

namespace WindowsApplication1
{
    public class DXMenuItemWrapper
    {

        private DXMenuItem _Item;
        public DXMenuItemWrapper(DXMenuItem item)
        {
            _Item = item;
        }

        public override string ToString()
        {
            return _Item.Caption;
        }

        public void GenerateClickEvent()
        {
            _Item.GenerateClickEvent();
        }
    }
}

File: VB/Custom menu/DXMenuItemWrapper.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraEditors
Imports DevExpress.Utils.Menu

Namespace WindowsApplication1
	Public Class DXMenuItemWrapper

		Private _Item As DXMenuItem
		Public Sub New(ByVal item As DXMenuItem)
			_Item = item
		End Sub

		Public Overrides Function ToString() As String
			Return _Item.Caption
		End Function

		Public Sub GenerateClickEvent()
			_Item.GenerateClickEvent()
		End Sub
	End Class
End Namespace
